health-services/transformer/src/main/java/org/egov/transformer/service/ProjectService.java

```diff
@@ -447,36 +447,35 @@ private MdmsCriteriaReq getMdmsRequest(RequestInfo requestInfo, String tenantId,
         mdmsCriteriaReq.setRequestInfo(requestInfo);
         return mdmsCriteriaReq;
     }
-    // TODO commented below code as staff model is changed and this search call is not being used as of now
-//    public ProjectStaff searchProjectStaff(String userId, String tenantId) {
-//
-//        ProjectStaffSearchRequest request = ProjectStaffSearchRequest.builder()
-//                .requestInfo(RequestInfo.builder()
-//                        .userInfo(User.builder()
-//                                .uuid("transformer-uuid")
-//                                .build())
-//                        .build())
-//                .projectStaff(ProjectStaffSearch.builder().staffId(userId).tenantId(tenantId).build())
-//                .build();
-//
-//        ProjectStaffBulkResponse response;
-//        try {
-//            StringBuilder uri = new StringBuilder();
-//            uri.append(transformerProperties.getProjectHost())
-//                    .append(transformerProperties.getProjectStaffSearchUrl())
-//                    .append("?limit=").append(transformerProperties.getSearchApiLimit())
-//                    .append("&offset=0")
-//                    .append("&tenantId=").append(tenantId);
-//            response = serviceRequestClient.fetchResult(uri,
-//                    request,
-//                    ProjectStaffBulkResponse.class);
-//        } catch (Exception e) {
-//            log.error("Error while fetching project staff list {}", ExceptionUtils.getStackTrace(e));
-//
-//            return null;
-//        }
-//        return !response.getProjectStaff().isEmpty() ? response.getProjectStaff().get(0) : null;
-//    }
+
+    public ProjectStaff searchProjectStaff(String userId, String tenantId) {
```

Comment:
search of project staff by userid list and tenant id is there, so why is this required?